<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="maximumSize">
       <size>
        <width>477</width>
        <height>32</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="searchFiled">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBase_de_donn_es">
    <property name="title">
     <string>Base de données</string>
    </property>
    <addaction name="actionMettre_jour"/>
    <addaction name="actionSave_XML"/>
    <addaction name="actionPr_f_rences"/>
   </widget>
   <addaction name="menuBase_de_donn_es"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_XML"/>
   <addaction name="actionMettre_jour"/>
   <addaction name="actionPr_f_rences"/>
  </widget>
  <action name="actionMettre_jour">
   <property name="text">
    <string>Mettre à jour</string>
   </property>
  </action>
  <action name="actionSave_XML">
   <property name="text">
    <string>save XML</string>
   </property>
  </action>
  <action name="actionPr_f_rences">
   <property name="text">
    <string>Préférences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
